examples/c-guest/guest/.gitignore

```diff
@@ -0,0 +1 @@
+out/
```

Comment:
Oh, surprised to see this in another `.gitignore `file. I guess either will work

Reply:
yeah maybe just a habit, but prefer to have dir specific ignores to be more resilient to refactors 🤷 Happy to switch to just using one though, if you prefer!